pages/games/[id]/index.js

```diff
@@ -0,0 +1,100 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import styled from "styled-components";
+
+export default function DetailsPage() {
+  const router = useRouter();
+  const { id } = router.query;
+  const {
+    data: game,
+    isLoading,
+    error,
+  } = useSWR(`/api/games?endpoint=/thing?id=${id}`);
+  console.log(game);
+  if (isLoading || error) return <h2>Loading...</h2>;
+
+  return (
+    <>
+      <H1>
+        Game<Span>dalf</Span>
+      </H1>
```

Comment:
in the future it might be beneficial to move this in a Layout component that you use for every page. But for now its fine.